po/pt_BR.po

```diff
@@ -312,11 +314,11 @@ msgstr "Nenhum"
 
 #: ../src/widgets.vala:1061 ../src/widgets.vala:1070
 msgid "Supported audio files"
-msgstr ""
+msgstr "Formatos de áudio suportados"
 
 #: ../data/gnome-pomodoro.desktop.in.h:2
 msgid "Manage your time and tasks"
-msgstr ""
+msgstr "Gerencie seu tempo e suas tarefas"
 
 #: ../data/gnome-pomodoro.desktop.in.h:3
 msgid "time;timer;tasks;manage;organize;"
```

Comment:
Should this be translated? Do will Gnome Shell recognize all locales **simultaneously?**